presidio-analyzer/presidio_analyzer/predefined_recognizers/ip_recognizer.py

```diff
@@ -1,3 +1,4 @@
+import ipaddress
```

Comment:
Since this package is quite old, should we consider taking the relevant code pieces (with the license + link to original) instead of importing it? how complex would that be in your view?

Reply:
@omri374 
Its a core library, not an external dependency